proto/docs/schema/README.md

```diff
@@ -0,0 +1,688 @@
+# Protocol Documentation
```

Comment:
In initial comment I meant to place it at `/proto/README.md` but I see we already have other documentation in proto context.

In light of that probably best name is `/proto/docs/schema.md`, wdyt ?

Reply:
That's fine with me, I'll make the change.